### ISSUE TITLE
Packaging [ubuntu]: missing 'Maintainer' field

### ISSUE THREAD

**Issue by edsantiago** [containers/podman#14849]
Seen while debugging another issue:
```console
# dpkg -l runc
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 911 package 'buildah':
 missing 'Maintainer' field
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 1428 package 'containers-common':
 missing 'Maintainer' field
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 1586 package 'crun':
 missing 'Maintainer' field
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 2667 package 'fuse-overlayfs':
 missing 'Maintainer' field
dpkg-query: warning: parsing file '/var/lib/dpkg/status' near line 16341 package 'skopeo':
 missing 'Maintainer' field
```
Obviously this is an issue with the respective packages, not podman, but I'm filing here anyway as a placeholder. @lsm5 is this an easy one to fix?

**Comment by lsm5** [comment]
should be. It's obviously coming from the `deb packages using fedora spec files` repo, I'm just curious to know where you're seeing this.

Also, this is just an annoying warning message, shouldn't really block installation

**Comment by edsantiago** [comment]
Yes, it's just a warning that doesn't affect anything, at least in CI-land: if these packages are used by humans, they might find the warnings objectionable.

I found it by playing in an ubuntu VM using the Cirrus rerun-in-term option. This is not something we would otherwise see in CI.

**Comment by lsm5** [comment]
ack, we will start seeing it in CI once the ubuntu 22.04 LTS PR is merged.

**Comment by rhatdan** [comment]
@lsm5 has this been fixed?

**Comment by lsm5** [comment]
> @lsm5 has this been fixed?

I think so, but let me verify manually sometime this week and close it out.

**Comment by edsantiago** [comment]
Looks fixed:
```
# dpkg -l runc
Desired=Unknown/Install/Remove/Purge/Hold
| Status=Not/Inst/Conf-files/Unpacked/halF-conf/Half-inst/trig-aWait/Trig-pend
|/ Err?=(none)/Reinst-required (Status,Err: uppercase=bad)
||/ Name           Version        Architecture Description
+++-==============-==============-============-=================================
ii  runc           1.1.0-0ubuntu1 amd64        Open Container Project - runtime
```
